mex/extractors/datenkompass/transform.py

```diff
@@ -67,12 +114,19 @@ def get_title(item: MergedActivity) -> list[str]:
 
 
 def get_vocabulary(
-    entries: Iterable[Theme | BibliographicResourceType],  # "list doesn't accept '|' "
+    entries: Iterable[
+        Theme
+        | BibliographicResourceType
+        | Frequency
+        | License
+        | ResourceCreationMethod
+        | ResourceTypeGeneral
+    ],  # "list doesn't work well with '|' "
```

Comment:
```suggestion
entries: list[_VocabularyT],
```
add after the imports:
```
_VocabularyT = TypeVar(
    "_VocabularyT",
    Theme,
    BibliographicResourceType,
    Frequency,
    License,
    ResourceCreationMethod,
    ResourceTypeGeneral,
)
```